downstream/modules/aap-hardening/con-rbac.adoc

```diff
@@ -7,17 +7,21 @@
 
 [role="_abstract"]
 
-As an administrator, you can use the Role-Based Access Controls (RBAC) built into {ControllerName} to delegate access to server inventories, organizations, and more. Administrators can also centralize the management of various credentials, allowing end users to leverage a needed secret without ever exposing that secret to the end user. RBAC controls allow the controller to help you increase security and streamline management.
+As an administrator, you can use the _Role-Based Access Controls_ (RBAC) built into the {GatewayStart} to delegate access to server inventories, organizations, and more. 
+Administrators can also centralize the management of various credentials, enabling end users to use a needed secret without ever exposing that secret to the end user. 
+RBAC controls allow {PlatformNameShort} to help you increase security and streamline management.
 
-RBAC is the practice of granting roles to users or teams. RBACs are easiest to think of in terms of Roles which define precisely who or what can see, change, or delete an “object” for which a specific capability is being set.
+RBAC is the practice of granting roles to users or teams. 
+RBAC is easiest to think of in terms of Roles which define precisely who or what can see, change, or delete an “object” for which a specific capability is being set.
 
-There are a few main concepts that you should become familiar with regarding {ControllerName}'s RBAC design–roles, resources, and users. Users can be members of a role, which gives them certain access to any resources associated with that role, or any resources associated with “descendant” roles.
+There are a few main concepts that you should become familiar with regarding {PlatformNameShort}'s RBAC design–roles, resources, and users. 
+Users can be members of a role, which gives them certain access to any resources associated with that role, or any resources associated with “descendant” roles.
 
 A role is essentially a collection of capabilities. Users are granted access to these capabilities and the controller’s resources through the roles to which they are assigned or through roles inherited through the role hierarchy.
 
 Roles associate a group of capabilities with a group of users. All capabilities are derived from membership within a role. Users receive capabilities only through the roles to which they are assigned or through roles they inherit through the role hierarchy. All members of a role have all capabilities granted to that role. Within an organization, roles are relatively stable, while users and capabilities are both numerous and may change rapidly. Users can have many roles.
```

Comment:
This passage repeats some of the info and wording in the sentence above it. If this is deliberate, ignore this comment